src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/ValidatorService.cs

```diff
@@ -160,40 +160,53 @@ private async Task<Either<List<DataImportError>, Unit>> ValidateCsvFileType(
 
             var reader = new MetaDataFileReader(csvHeaders);
 
+            var hasDefaultColumn = csvHeaders.Contains(MetaColumns.filter_default.ToString());
+            var filterGroups = 0;
+
             await CsvUtils.ForEachRow(fileStreamProvider, (cells, index, _) =>
             {
                 totalRows++;
-                
+
                 if (cells.Count != csvHeaders.Count)
                 {
                     var errorCode = MetaFileHasInvalidNumberOfColumns;
                     errors.Add(new DataImportError($"Error at data row {index + 1}: {errorCode.GetEnumLabel()}"));
                 }
-                
+
                 try
                 {
-                    reader.GetMetaRow(cells);
+                    var row = reader.GetMetaRow(cells);
+
+                    if (!string.IsNullOrWhiteSpace(row?.FilterGroupingColumn))
+                    {
+                        filterGroups++;
+                    }
                 }
                 catch (Exception e)
                 {
                     errors.Add(new DataImportError($"error at data row {index + 1}: {MetaFileHasInvalidValues.GetEnumLabel()} : {e.Message}"));
                 }
             });
 
+            if (hasDefaultColumn && filterGroups > 1)
+            {
+                errors.Add(new DataImportError("The filter_default column cannot be used in conjunction with filter hierarchies"));
```

Comment:
Similar to earlier comment - would suggest leaving a comment here on why we're currently doing this and leave a TODO referencing the relevant ticket that should address this